e2e/login.setup.ts

```diff
@@ -0,0 +1,21 @@
+import { test as setup, expect } from '@playwright/test';
+import 'dotenv/config';
+import { STORAGE_STATE } from '../playwright.config';
+
+setup('Login', async ({ page }) => {
+    await page.goto('https://localhost:19888/');
```

Comment:
once baseurl has been added use '/' here instead of 'https://localhost:19888/'

Reply:
Replaced with /